draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -508,56 +508,27 @@ hop(AS(i), AS(j)) =  / Else, "Provider+" if the VAP-SPAS entry
-      For any route with an Unknown AS_PATH, the cause of the Unknown state SHOULD be logged for monitoring and diagnostic purposes.
-      The cause of the Unknown state can be in the form of listing the AS hops which were evaluated by the hop-check function to be "No Attestation" or "Not Provider+".
-    </t>
-  </section>
-  <!--
-  <section title="Verification and Mitigation at Egress eBGP Router" anchor="egress">
-    <t>	
-      <strong>Verification:</strong> The procedures for AS_PATH verification (<xref target="verif"/>) are also applicable at egress eBGP routers for preventing an AS from sending routes with Invalid AS_PATH to its eBGP neighbors (see <xref target="RFC8893"/> for a comparable idea for the RPKI-ROV case).
-      An egress eBGP router MUST add the AS of the router's BGP configuration (see <xref target="RFC8481"/> <xref target="RFC8893"/>) to the received AS_PATH (received in iBGP) and then apply the path verification procedures.
-      When redistributing into BGP from any source (e.g., IGP, iBGP, or from static or connected routes), there is no AS_PATH in the input route.
-      In such cases, the egress eBGP router MUST use the AS of the router's BGP configuration to form the AS_PATH for verification (see <xref target="RFC8481"/>).
-   </t>
-   <t>
-     <strong>Mitigation:</strong> Again, mitigation recommendations are provided here with the understanding that the deployed mitigation policy is set by network operator discretion.
-     If the outcome of applying the upstream algorithm (<xref target="Upflow"/>) is Invalid, then the route SHOULD NOT be propagated to a transit provider, lateral peer, RS (local AS has RS-client role), or RS-client (local AS has RS role).
-     If the outcome of applying the downstream algorithm (<xref target="Downflow"/>) is Invalid, then the route SHOULD NOT be propagated to an eBGP neighbor regardless of its BGP role (<xref target="role"/>).
-   </t>
-  </section>
-  -->
-   <section title="Only to Customer (OTC) Attribute" anchor="otc">
-   <t>
-     While the ASPA-based AS_PATH verification method (<xref target="ingress"/>) detects and mitigates route leaks that were created by preceding ASes listed in the AS_PATH, 
-	 it lacks the ability to prevent route leaks from occuring at the local AS. 
-	 The use of the Only to Customer (OTC) Attribute <xref target="RFC9234"/> fills in that gap. 
-	 The procedures utilizing the OTC Attribute set out in <xref target="RFC9234"/> complement those described in this document. 
-	 Implementation of those procedures in addition to ASPA-based AS_PATH verification is encouraged.
-   </t>
-   </section>
+    <section anchor="ingress" title="Ingress eBGP Router vs. Internal BGP">
+      <t>ASPA can run on any BGP router. However, procedures for ASPA-based AS_PATH verification and mitigation described in this document are NOT RECOMMENDED for use on eBGP links internal to the Confederation.</t>
```

Comment:
Can ASPA run on any BGP router? I think not - because ASPA depends on the configuration of roles and there is no IBGP role. So an IBGP router cannot do ASPA

Reply:
typo. we probably meant "ASPA can run on any eBGP router." -- would you be fine with this?

Reply:
ASPA SHOULD run on any eBGP router. I find the title misleading: `Ingress eBGP Router vs. Internal BGP` since this is about BGP confederations. Isn't this mostly about running ASAP validation on inter-AS border routers and it should not be run inside an AS (including confederations)?

Reply:
agree @cjeker. what about the section titles: (1) "Where to run ASPA?" or (2) "Running ASPA on Inter-AS Border Routers" ?